src/generated/quicktools/CveRecordV5.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * This file was automatically generated https://app.quicktype.io/?l=ts.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and regenerate this file.
+*/
+// To parse this data:
+//
+//   import { Convert, CveRecordV5 } from "./file";
+//
+//   const cve5 = Convert.toCve5(json);
+
+export interface CveRecordV5 {
+  containers?: Containers;
+  cveMetadata?: CveMetadata;
+  dataType?: string;
+  dateVersion?: string;
+  [property: string]: any;
+}
+
+export interface Containers {
+  cna: Cna;
+  [property: string]: any;
+}
+
+export interface Cna {
```

Comment:
For a 'rejected' record, the 'cnaContainer' also contains a '[rejectedReasons](https://github.com/CVEProject/cve-schema/blob/master/schema/v5.0/CVE_JSON_5.0_schema.json#L568)'. Are we considering that at all?

Reply:
This is going to take some more investigation.  The generator I used isn't responding in the same way it did when I first used it back in the spring.  Looking through the code, the CVEs are read in as JSON, and written out as JSON, so this isn't a critical problem for now.  I have verified this by checking the CVEs in CVEProject/cvelistV5, and the REJECTED records are correct.  I have also opened this as an issue for further investigation:  https://github.com/CVEProject/cvelist-bulk-download/issues/5

Reply:
I think given that this bug is not affecting the bulk download process, since the github action is basically just referencing the CveMetadata portion of the JSON, and is writing the body of the JSON out verbatim (plus prettying), this bug can be triaged for a later sprint.